server/serverAPI/controller/AuthenticationController.ts

```diff
@@ -76,7 +76,14 @@ export default class AuthenticationController extends BaseController {
 
             let token = this.tokenCreator.sign(identification, this.timeoutTimeInSeconds);
 
-            return this.sendSuccess(200, res, { accessToken: token });
+            if (req.query.includeInfo === 'true') {
```

Comment:
Is it possible to use a boolean instead of a string?

Reply:
no, because query returns strings...